src/pages/ScholarshipForm.jsx

```diff
@@ -25,43 +68,70 @@ function ScholarshipForm() {
     setFormFieldStates({ ...formFieldStates, minAmount, maxAmount });
   }
 
-  function requiredTextField(id, label) {
-    return (
-      <TextField
-        id={id}
-        label={label}
-        required
-        fullWidth
-        value={formFieldStates[id]}
-        onChange={updateFn(id)}
-      />
-    );
-  }
-
   return (
     <Container maxWidth="md">
       <form>
         <h1>Submit a Scholarship</h1>
-        {requiredTextField('name', 'Scholarship Name')}
+        <div>
+          <TextField
+            id="name"
+            label="Scholarship Name"
+            error={formFieldErrors.name}
+            helperText={formFieldErrors.name ? 'Please enter a name' : ' '}
+            className={classes.textFieldStyle}
+            required
+            value={formFieldStates.name}
+            onChange={updateFn('name')}
+          />
+        </div>
         <DatePicker
           id="deadline"
           label="Deadline"
+          error={formFieldErrors.deadline}
+          helperText="Please enter a valid date"
```

Comment:
It's because before I had `helperText={error ?  HelperText : ' ']`
So since the error is false before clicking the submit button, the `helperText` doesn't show

Reply:
Ahh alright yeah that's a little confusing considering what `helperText` means in Material UI Form components